manual-tests/test-aarch64-debian.dockerfile.broken

```diff
@@ -0,0 +1,118 @@
+# Test MPItrampoline on various CPU architectures
```

Comment:
what's the stuff in `manual-tests`?

Reply:
These are scripts that install Julia and `MPI.jl` on various systems (either Docker containers or real-world HPC systems) to test that `MPI.jl` works with the system MPI libraries there.

Reply:
including the `.broken` one?

I'm not so keen on having system-specific files in here, as there is no way to ensure they are kept up-to-date.

Reply:
I removed the manual tests.